libs/go/sia/host/hostdoc/raw/raw.go

```diff
@@ -9,7 +11,11 @@ type Doc struct {
 	Services          string   `json:"services,omitempty"`
 	Profile           string   `json:"profile"`
 	ProfileRestrictTo string   `json:"profile_restrict_to,omitempty"`
+	AccountId         string   `json:"account_id,omitempty"`
+	ProjectNumber     string   `json:"project_number,omitempty"`
 	Uuid              string   `json:"uuid,omitempty"`
 	Ip                []string `json:"ip,omitempty"`
 	Zone              string   `json:"zone,omitempty"`
+	// stop/start an instance will update the LaunchTime
+	LaunchTime *time.Time `json:"launch_time,omitempty"`
```

Comment:
please have the fields aligned with the rest of the attributes

Reply:
Adding the comment in a separate line will break the auto-align into this format. I will add the comment in the same line of the field.